index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-ca">
   <head>
     <!-- Set the page title -->
-    <title>Assignment 02</title>
+    <title>Homi-site - Assignment 02</title>
 
```

Comment:
Missing the scale meta tag declaration that was included in the assignment boilerplate